bin/sync-assets.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+declare bindir
+bindir="$(realpath "$(dirname "$0")")"
+cd "$bindir" || exit 1
+
+while read -r dest; do
+    echo "Syncing $dest..."
+    rsync \
```

Comment:
Did something go wrong? I see, for some test exercise , the contents of the assets appears under the exercise's lib dir. For these exercise I see 2 makefiles: 

* all-fail
* empty-file
* enotimplemented-multi
* enotimplemented-single
* partial-fail
* success
* syntax-error
* user-output

unhandled-exception is the one that looks correct.